### PR TITLE
Allows user to change color theme of page and fixes code indentation.

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,10 +17,12 @@
...
-      ['I have a scar on the back of my head from when I fell backwards out of a chair when I was in 5th grade.',
-       'I backpacked through Portugal with a Catalan Troop.', 
-       'I have 6 pets, 5 cats and 1 dog.', 
-       'I forgot I was in the orchestra pit for "South Pacific."'];
+    ['I have a scar on the back of my head from when I fell backwards out of a chair when I was in 5th grade.',
+      'I backpacked through Portugal with a Catalan Troop.',
+      'I have 6 pets, 5 cats and 1 dog.',
+      'I forgot I was in the orchestra pit for "South Pacific."',
+      'I still have a baby tooth.',
+      "I've baked chocololate chip cookies for my entire highschool more than once."];
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/script.js]: The style guide says to use single quotes everywhere in JavaScript, so probably just escape the internal quote. It also says you can use template strings, so maybe look into that: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

**Comment by acmcarther** [comment]
This PR includes the content of the other PRs so I'm not sure what's different.

Consider using separate feature branches instead of piling everything into one branch.

**Review by jalexanderqed — COMMENTED** [review]
It looks like there a still a lot of tabs, but if it's a lot of work to fix that, maybe make it a separate pull request (that just fixes the tabs).

**Comment by jalexanderqed** [comment]
> This PR includes the content of the other PRs so I'm not sure what's different.
> 
> Consider using separate feature branches instead of piling everything into one branch.

Meagan and I discussed this and decided it would be best to just make everything one big CL (thus abandoning #4) for now. We'll work on making things more refined in future.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (meagancheese) [reply portfolio/src/main/webapp/script.js]: I thought I made it so that each new indent was two spaces. Is it possible that it is just super indented?

Comment (meagancheese) [reply portfolio/src/main/webapp/script.js]: Oh, so the quotation marks should line up with the first line? Ok.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,33 +1,101 @@
...
+    <p>Click here to learn a random fact about me:</p>
+    <button onclick="addRandomFact()">Random Fact</button>
+    <div id="random-fact-container"></div>
+      <h2>My Favorites</h2>
+      <p>Click here to learn one of my favorite things:</p>
+      <button onclick="addRandomFavorite()">Favorite Thing</button>
+      <div id="random-favorite-container"></div>
+    <h2>My Background</h2>
+    <p>I started my education at a public school called Brooklands Elementary School that was within walking
+           distance of my house. My favorite thing about going to Brooklands was that all of
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/index.html]: Please re-indent this section to match the `>` in `<p>`. Example here: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Generally, everything should be indented two spaces from the enclosing block.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,33 +1,101 @@
...
+          <option value="blue">Blue</option>
+          <option value="red">Red</option>
+          <option value="green">Green</option>
+          <option value="pink">Pink</option>
+          <option value="yellow">Yellow</option>
+          <option value="purple" selected>Go Back to Purple</option>
+        </select>
+    <h2>My Music</h2>
+    <p>Music is a huge part of who I am. When I was a toddler my family would go to the Detroit Symphony
+        Orchestra's Children's Concert Series and my mom tells me stories of how
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/index.html:56]: Re-indent this block in the same manner.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,33 +1,101 @@
...
-      <img src="/images/Selfie.jpg" alt="Meagan Selfie" width="300" height="400" class="center">
+
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <div id="content">
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/index.html:11]: Everything (so most of the document) inside this `<div>` should be indented one more time (two spaces).

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,33 +1,101 @@
...
+  <h1>Meagan Konst's Portfolio</h1>
+  <img src="/images/Selfie.jpg" alt="Meagan Selfie" width="300" height="400" class="center">
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/index.html]: All of the `` because they have no closing tag and are therefore unclosed currently.

**Review by jalexanderqed — APPROVED** [review]
Looks good! Just some nit-picky style fixes.

**Pull request merged**